app/src/main/kotlin/com/simplemobiletools/clock/helpers/Constants.kt

```diff
@@ -26,18 +26,23 @@ const val WAS_INITIAL_WIDGET_SET_UP = "was_initial_widget_set_up"
 const val TABS_COUNT = 4
 const val EDITED_TIME_ZONE_SEPARATOR = ":"
 const val ALARM_ID = "alarm_id"
+const val NOTIFICATION_ID = "notification_id"
+const val ALARM_TIME = "alarm_time"
 const val DEFAULT_ALARM_MINUTES = 480
 const val DEFAULT_MAX_ALARM_REMINDER_SECS = 300
 const val DEFAULT_MAX_TIMER_REMINDER_SECS = 60
 const val SIMPLE_PHONE = "Simple_Phone"
 const val ALARM_NOTIFICATION_CHANNEL_ID = "Alarm_Channel"
+const val EARLY_ALARM_DISMISSAL_CHANNEL_ID = "Early Alarm Dismissal"
 
 const val PICK_AUDIO_FILE_INTENT_ID = 9994
 const val REMINDER_ACTIVITY_INTENT_ID = 9995
 const val OPEN_ALARMS_TAB_INTENT_ID = 9996
 const val OPEN_STOPWATCH_TAB_INTENT_ID = 9993
 const val OPEN_APP_INTENT_ID = 9998
+const val EARLY_ALARM_DISMISSAL_INTENT_ID = 9999
 const val ALARM_NOTIF_ID = 9998
+const val EARLY_ALARM_NOTIF_ID = 9999
```

Comment:
I dont think it is a good idea to reuse 9998 and 9999 twice, having different numbers is the point of having the constants here on 1 place

Reply:
resolved